tests/devices/test_default_qubit_autograd.py

```diff
@@ -273,23 +273,16 @@ def test_autograd_interface_gradient(self, operation, diff_method, tol):
         """Tests that the gradient of an arbitrary U3 gate is correct
         using the Autograd interface, using a variety of differentiation methods."""
         dev = qml.device("default.qubit.autograd", wires=1)
+        state = np.array(1j * np.array([1, -1]) / np.sqrt(2), requires_grad=False)

```

Comment:
Was this moved because `requires_grad=True` is the default within a quantum function?

Reply:
yep :(

Reply:
And shouldn't processing inside qnodes be even more possible now?

Reply:
Yep. I could have left this inside the QNode with the `requires_grad=False` explicit, but thought the test was just clearer having it here.

So, the `requires_grad=False` is _required_, but moving it was just me thinking it looked nicer! Sorry about the confusion.